cloudflare/resource_cloudflare_record.go

```diff
@@ -147,20 +198,26 @@ func resourceCloudFlareRecordRead(d *schema.ResourceData, meta interface{}) erro
 	d.Set("ttl", record.TTL)
 	d.Set("priority", record.Priority)
 	d.Set("proxied", record.Proxied)
-	d.Set("zone_id", zoneId)
+	d.Set("created_on", record.CreatedOn.Format(time.RFC3339Nano))
+	d.Set("data", expandStringMap(record.Data))
+	d.Set("modified_on", record.ModifiedOn.Format(time.RFC3339Nano))
+	d.Set("metadata", expandStringMap(record.Meta))
```

Comment:
For non-scalar values (Maps, Lists, Sets), please check the error:

```go
if err := d.Set("metadata", expandStringMap(record.Meta)); err != nil {
  log.Printf("[WARN] Error setting metadata: %s", err)  
}
```

Reply:
Applied in https://github.com/terraform-providers/terraform-provider-cloudflare/commit/f01532c4a37f894b2926b54bed9e5d21d19255cd